Library/Formula/binwalk.rb

```diff
@@ -66,6 +66,13 @@ def install
       bin.install Dir["#{libexec}/bin/*"]
       bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
     else
+      # Fix hard-coded path to Homebrew cellar.
+      inreplace "src/binwalk/core/C.py", "('/usr/local/Cellar/", "('#{HOMEBREW_CELLAR}/"
+
+      # Fix hard-coded extension of dynamic libraries. Otherwise both libraries
+      # "libcompress42.dylib" and "libtinfl.dylib" will not be installed.
+      inreplace "setup.py", "\"*.so\"", "\"*.dylib\""
```

Comment:
Can you please submit these patches to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!